lib/config.js

```diff
@@ -45,5 +45,11 @@ export let config = {
                       details.`,
         type: 'boolean',
         default: false,
+    },
+    ignoredFiles: {
```

Comment:
`ignoredExtensions` is a more accurate name for this config setting, since the value doesn't contain files, but extensions.

Reply:
renamed to `ignoredPatterns` since it now uses glob patterns.